identity/src/lib/rpc/server.rs

```diff
@@ -11,19 +11,21 @@ use helpers::rpc::{Error, RpcResult};
 use super::{models::*, service::IdentityService};
 use crate::authentication::oauth;
 use crate::config::Configuration;
-use crate::db::{models, Db};
+use crate::db::{models, DbPool};
 use crate::session::jwt::Jwt;
 
 #[derive(Clone)]
 pub struct IdentityServer {
     pub addr: SocketAddr,
     pub conf: Arc<Configuration>,
-    pub db: Arc<Db>,
+    pub db_pool: Arc<DbPool>,
```

Comment:
I was just thinking if any of this needs to be actually public? Probably not, right?

Reply:
If you do not have a constructor for the server then yes. I just wrote one and therefore changed it to private.